web/src/components/tasks/business-formation/name/NexusSearchBusinessNameStep.test.tsx

```diff
@@ -89,115 +117,96 @@ describe("<NexusSearchBusinessNameStep />", () => {
-    renderTask();
-    fillText("some unavailable name");
-    await searchAndGetValue({ status: "UNAVAILABLE" });
+    const page = await getPageHelper({});
+    page.fillText("Search business name", "some unavailable name");
+    await page.searchBusinessName({ status: "UNAVAILABLE" });
 
     fireEvent.click(screen.getByText(Config.nexusNameSearch.searchAgainButtonText));
-    fillText("My Cool Business");
-    await searchAndGetValue({ status: "AVAILABLE" });
+    page.fillText("Search business name", "My Cool Business");
+    await page.searchBusinessName({ status: "AVAILABLE" });
     expect(currentUserData().formationData.formationFormData.businessName).toEqual("My Cool Business");
   });
 
   it("sets needsDbaName in profile to true when unavailable", async () => {
-    renderTask();
-    fillText("My Cool Business");
-    await searchAndGetValue({ status: "UNAVAILABLE" });
+    const page = await getPageHelper({});
+    page.fillText("Search business name", "My Cool Business");
+    await page.searchBusinessName({ status: "UNAVAILABLE" });
     expect(currentUserData().profileData.nexusDbaName).toEqual("");
     expect(currentUserData().profileData.needsNexusDbaName).toEqual(true);
     expect(currentUserData().profileData.businessName).toEqual("My Cool Business");
   });
 
   it("does not save business name if designator error", async () => {
-    renderTask();
-    fillText("My Cool Business");
-    await searchAndGetValue({ status: "DESIGNATOR_ERROR" });
+    const page = await getPageHelper({});
+    page.fillText("Search business name", "My Cool Business");
+    await page.searchBusinessName({ status: "DESIGNATOR_ERROR" });
     expect(userDataWasNotUpdated()).toEqual(true);
   });
 
   it("shows DBA name when unavailable", async () => {
-    renderTask();
-    fillText("My Cool Business");
-    await searchAndGetValue({ status: "UNAVAILABLE" });
+    const page = await getPageHelper({});
+    page.fillText("Search business name", "My Cool Business");
+    await page.searchBusinessName({ status: "UNAVAILABLE" });
     expect(screen.getByText(markdownToText(Config.nexusNameSearch.dbaNameHeader))).toBeInTheDocument();
     expect(screen.getByText(Config.nexusNameSearch.dbaNameSearchSubmitButton)).toBeInTheDocument();
     expect(
       mockAnalytics.event.business_formation_dba_name_search_field.appears.dba_name_search_field_appears
     ).toHaveBeenCalled();
   });
 
-  it("shows DBA search immediately if needsDbaName is true", async () => {
-    useMockUserData({
-      profileData: generateProfileData({
-        businessName: "some cool name",
-        needsNexusDbaName: true,
-      }),
-    });
-    mockSearchReturnValue({ status: "UNAVAILABLE" });
-    renderTask();
-    await waitFor(() => {
-      expect(screen.getByText(markdownToText(Config.nexusNameSearch.dbaNameHeader))).toBeInTheDocument();
+  it("does not show DBA search immediately if it is needsNexusDbaName is false in profile", async () => {
```

Comment:
Why the deletion of the corresponding "if true" test?

Reply:
Bad copy paste job. It's back in there.